scripts/install/SetMediaManager.php

```diff
@@ -0,0 +1,82 @@
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+namespace oat\taoMediaManager\scripts\install;
+
+use common_report_Report;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\extension\InstallAction;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\taoItems\model\event\ItemRemovedEvent;
+use oat\taoItems\model\event\ItemUpdatedEvent;
+use oat\taoMediaManager\model\fileManagement\FlySystemManagement;
+use oat\taoMediaManager\model\fileManagement\FileManagement;
+use oat\tao\model\media\MediaService;
+use oat\taoMediaManager\model\MediaSource;
+use oat\taoMediaManager\model\relation\event\MediaRelationListener;
+use oat\taoMediaManager\model\relation\event\MediaRemovedEvent;
+use oat\taoMediaManager\model\relation\event\MediaSavedEvent;
+use oat\taoMediaManager\model\relation\repository\MediaRelationRepositoryInterface;
+use oat\taoMediaManager\model\relation\repository\rdf\map\RdfItemRelationMap;
+use oat\taoMediaManager\model\relation\repository\rdf\map\RdfMediaRelationMap;
+use oat\taoMediaManager\model\relation\repository\rdf\RdfMediaRelationRepository;
+
+class SetMediaManager extends InstallAction
+{
+    public function __invoke($params)
+    {
+        $fsService = $this->getServiceManager()->get(FileSystemService::SERVICE_ID);
+        $fsService->createFileSystem('mediaManager');
+
+        $this->getServiceManager()->register(FileSystemService::SERVICE_ID, $fsService);
+        $this->getServiceManager()->register(
+            FileManagement::SERVICE_ID,
+            new FlySystemManagement(
+                [
+                    FlySystemManagement::OPTION_FS => 'mediaManager'
+                ]
+            )
+        );
+        $this->getServiceManager()->register(
+            MediaRelationRepositoryInterface::SERVICE_ID,
+            new RdfMediaRelationRepository(
+                [
+                    RdfMediaRelationRepository::MAP_OPTION => [
+                        new RdfItemRelationMap(),
+                        new RdfMediaRelationMap()
+                    ]
+                ]
+            )
+        );
+
+        /** @var EventManager $eventManager */
+        $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+        $eventManager->attach(ItemUpdatedEvent::class, [MediaRelationListener::class, 'whenItemIsUpdated']);
+        $eventManager->attach(ItemRemovedEvent::class, [MediaRelationListener::class, 'whenItemIsRemoved']);
+        $eventManager->attach(MediaRemovedEvent::class, [MediaRelationListener::class, 'whenMediaIsRemoved']);
+        $eventManager->attach(MediaSavedEvent::class, [MediaRelationListener::class, 'whenMediaIsSaved']);
+
+        $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
+
+        MediaService::singleton()->addMediaSource(new MediaSource());
+
+        return common_report_Report::createSuccess('Successfully installed');
```

Comment:
I would suggest to add some context to the message
```suggestion
        return common_report_Report::createSuccess('MediaManager successfully installed');
```